db/migrate/20201223013917_create_scratchpad_page.rb

```diff
@@ -0,0 +1,12 @@
+class CreateScratchpadPage < ActiveRecord::Migration[6.1]
+  def change
+    create_table :scratchpad_pages do |t|
+      t.belongs_to :about, polymorphic: true
```

Comment:
Just checking: we are okay with the scratchpad not being "about" anything? What would be the use case for it to be null. A personal scratchpad for the user?

Reply:
No, this should be `null: false`. Good catch.